test/spec/sparkpost.spec.js

```diff
@@ -248,14 +255,41 @@ describe('SparkPost Library', function() {
         done();
       });
     });
+
+    it('should return a parsed JSON object', function(done) {
+      nock('https://test.sparkpost.com')
+        .get('/test')
+        .reply(200, '{ "ok": true }');
+
+      var options = {
+        method: 'GET'
+        , uri: 'https://test.sparkpost.com/test'
+      };
+
+      client.request(options, function(err, data) {
+        expect(data.body).to.not.be.a('string');
```

Comment:
Belt and suspenders. Also I didn't know if it was like "a function is an object, a string is an object, I'm javascript and I treat everything as an object"